_posts/2019/2019-03-10-dont-use-instance-variables-in-partials.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Don't use instance variables in partials
+description: Containment is key
+layout: article
+category: ruby
+image:
+  base: '2019/dont-use-instance-variables-in-partials'
+  alt: 'Partial eclipse'
+  source: "https://unsplash.com/photos/NbgQfUvKFE0"
+  credit: "Mark Tegethoff"
+---
+
+Rails’ view architecture is a flexible and magical place. Perhaps too much.
+
+Instance variables from controllers, like `@whatever`, are magically available in view templates. Inferred template names. A global namespace of helpers. For some it's too much, but most of us accept the “magic” and muddle through any strangeness.
+
+
+## Instead of…
+
+…using the global variables from your template inside your partials, pass the values as local variables.
```

Comment:
I think remove the '...pass the values...' part of this sentence, as I got confused thinking the below was the solution.